crates/configuration/src/parachain.rs

```diff
@@ -23,14 +24,24 @@ pub struct ParachainConfig {
     id: u32,
 
     /// Chain to use (use None if you are running adder-collator or undying-collator).
-    chain: Option<Chain>,
+    chain: Chain,
```

Comment:
If we will use an empty string as no-chain, used for example with adder/undying collators, we should document the behavior.
Thx!